build-tests/api-extractor-test-05/config/api-extractor.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "mainEntryPointFilePath": "<projectFolder>/lib/index.d.ts",
+  "apiReport": {
+    "enabled": true
+  },
+  "docModel": {
+    "enabled": true,
+    "apiJsonFilePath": "<projectFolder>/etc/<unscopedPackageName>.api.json"
+  },
+  "dtsRollup": {
```

Comment:
Where is the dts rollup? Is it purposefully not added as `dist/` is ignored by Git or something?

Reply:
Yeah the git is structured to ignore the `dist` directory; however it is 1:1 (spaces included) of our index.d.ts with an additional `export { }` appended to the end